test/config/100-test-namespace.yaml

```diff
@@ -16,3 +16,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: serving-tests
+  labels:
+    istio.io/dataplane-mode: ambient
```

Comment:
I'm assuming we need the label on the net-istio controller's namespace?

So that probing happens over the ambient mesh?

Reply:
https://github.com/knative-sandbox/net-istio/blob/main/test/config/100-serving-namespace.yaml

Reply:
Thank you. I missed it. Updated.